src/ServiceInsight.Tests/AutoApprover.cs

```diff
@@ -0,0 +1,41 @@
+﻿namespace ServiceInsight.Tests
+{
+    using System.IO;
+    using ApprovalTests.Core;
+
+    class AutoApprover : IReporterWithApprovalPower
```

Comment:
Yes I want to keep this class. It's an easy way to approve all the tests, especially when they've changed.